composer.json

```diff
@@ -16,10 +16,6 @@
         "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer:^2.7",
         "bats": "TERMINUS_PLUGINS_DIR=.. PATH=$HOME/bin:$PATH bats -p -t tests",
```

Comment:
I don't think this will work with Terminus 3.  Might want to remove `TERMINUS_PLUGINS_DIR`.

Reply:
Definitely. Done